staging/src/k8s.io/api/apps/v1/types.go

```diff
@@ -255,7 +255,8 @@ type StatefulSetSpec struct {
 	// policy allows the lifecycle to be altered, for example by deleting persistent
 	// volume claims when their stateful set is deleted, or when their pod is scaled
 	// down. This requires the StatefulSetAutoDeletePVC feature gate to be enabled,
-	// which is alpha.  +optional
+	// which is alpha.
```

Comment:
It's a beta field. https://github.com/kubernetes/enhancements/issues/1847

Reply:
Fixed :)